sample/F8/tracks

```diff
@@ -0,0 +1,202 @@
+[
+   {
+      "id":1,
+      "name":"ADS & MONETIZATION",
+      "color":"#BF532E",
+      "description":null,
+      "location":"C",
```

Comment:
@niranjan94 we have an [issue](https://github.com/fossasia/open-event-android/issues/1633) about description of tracks in the android repo. Could you please confirm that we should remove track description. While location of tracks don't make sense, track description could be used I believe.